c/driver/postgresql/copy/writer.h

```diff
@@ -477,7 +474,7 @@ class PostgresCopyTimestampFieldWriter : public PostgresCopyFieldWriter {
       return ADBC_STATUS_INVALID_ARGUMENT;
     }
 
-    if (value < std::numeric_limits<int64_t>::min() + kPostgresTimestampEpoch) {
+    if (value < (std::numeric_limits<int64_t>::min)() + kPostgresTimestampEpoch) {
```

Comment:
This is for avoiding name conflict on Windows.
`windef.h` defines `min` macro. It conflicts with `std::numeric_limits::min()`. If we use `(std::numeric_limits::min)()`, we can avoid the `min` macro expansion.